Makefile

```diff
@@ -75,11 +75,12 @@ test-playwright-webviews: setup-test-env install-test-dependencies install-depen
 # Usage: make test-playwright-e2e TEST_SUITE=@regression
 .PHONY: test-playwright-e2e
 test-playwright-e2e: setup-test-env install-test-dependencies install-dependencies
-	@if [ -n "$(TEST_SUITE)" ] && [ "$(TEST_SUITE)" != "" ]; then \
+	@if [ -n "$(TEST_SUITE)" ] && [ "$(TEST_SUITE)" != "" ] && [ "$(TEST_SUITE)" != "TEST_SUITE" ]; then \
```

Comment:
Unrelated fix to allow our Semaphore [task](https://semaphore.ci.confluent.io/projects/vscode/schedulers/56948747-fc5b-4ec9-b458-2b0847a90d83) to run without specifying a test suite/name. While testing with the existing task, not providing a `TEST_SUITE` value ends up setting the parameter value to ["TEST_SUITE"](https://semaphore.ci.confluent.io/jobs/23735b79-7fc1-4433-a697-c8c5852ed607#L493):
```
Running command: npx playwright test -c tests/e2e/playwright.config.ts tests/e2e -g TEST_SUITE
```